recipes/default.rb

```diff
@@ -17,17 +17,29 @@
 # limitations under the License.
 #
 
-# apt_repository 'lacework' do
-#   source "#{download_url}/latest/DEB/Ubuntu/16.04 xenial main"
-#   source "#{download_url}/latest/DEB/Debian/9 whatever main"
+dist = node['platform']
+pver =
+  if node['platform'] == 'debian'
+    node['platform_version'].to_i
+  else
+    node['platform_version'].to_f
```

Comment:
Is the `to_f` even necessary since this value gets converted back to a string when used below?

Reply:
It's to ensure it's only the major.minor and not the patch level. I could have split/joined, but this is faster. Yay dynamic types! 😁

Reply:
Did Ruby go to the MySQL school of data conversion/truncation?

```
$ irb
irb(main):001:0> '1.2.3'.to_f
=> 1.2
```